src/components/SubCategoryDetails/SubCategoryDetails.scss

```diff
@@ -0,0 +1,45 @@
+@import '~style/variables';
+
+.container {
```

Comment:
The charts fill the full width of the page. I believe everything should be contained to be max 1140px. I could be wrong though.

Reply:
You'd need to do this in here and for the hero chart.